src/init.sh

```diff
@@ -5,25 +5,60 @@ include "$KW_LIB_DIR/kwio.sh"
 
 KW_DIR='.kw'
 
+declare -gA options_values
+
 # This function is responsible for creating a local kworkflow.config based in a
 # template available in the etc directory.
 #
 # Returns:
-# In case of failure, this function return ENOENT.
+# In case of failure, this function returns ENOENT.
 function init_kw()
 {
   local config_file_template="$KW_ETC_DIR/kworkflow_template.config"
   local name="kworkflow.config"
 
+  init_parser "$@"
```

Comment:
In my view, `kw init` should only be invoked inside the kernel. Let's add `if ! is_kernel_root "$PWD"; then` before `init_parser`.